vlog4j-graal/src/main/java/org/semanticweb/vlog4j/graal/ImportedGraalQuery.java

```diff
@@ -0,0 +1,57 @@
+package org.semanticweb.vlog4j.graal;
+
+import static org.semanticweb.vlog4j.core.model.implementation.Expressions.makeAtom;
+import static org.semanticweb.vlog4j.core.model.implementation.Expressions.makeConjunction;
+import static org.semanticweb.vlog4j.core.model.implementation.Expressions.makePredicate;
+import static org.semanticweb.vlog4j.core.model.implementation.Expressions.makeRule;
+
+import java.util.List;
+import java.util.Set;
+
+import org.semanticweb.vlog4j.core.model.api.Atom;
+import org.semanticweb.vlog4j.core.model.api.Blank;
+import org.semanticweb.vlog4j.core.model.api.Conjunction;
+import org.semanticweb.vlog4j.core.model.api.Constant;
+import org.semanticweb.vlog4j.core.model.api.Predicate;
+import org.semanticweb.vlog4j.core.model.api.Rule;
+import org.semanticweb.vlog4j.core.model.api.Term;
+import org.semanticweb.vlog4j.core.model.api.Variable;
+import org.semanticweb.vlog4j.core.reasoner.Reasoner;
+import org.semanticweb.vlog4j.core.reasoner.implementation.VLogReasoner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import fr.lirmm.graphik.graal.api.core.ConjunctiveQuery;
+
+/**
+ * A utility class containing a {@link ConjunctiveQuery Graal ConjunctiveQuery}.
```

Comment:
Perhaps, instead of describing how to use the reasoner in a module that only depends on model entities, a more theoretical description can be provided: answering a program with a Graal ConjunctiveQuery (which can contain multiple atoms, and has non-answer variables) is equivalent to adding a rule to the program, which contains the conjunctiveQuery atoms as a body and a fresh atomic head over the conjunctive query variables, and then using the rule head as a query atom.

Reply:
Rewrote the javadoc accordingly.